cmdb/file_export/file_exporter.py

```diff
@@ -104,35 +109,38 @@ def get_object_by_id(self):
 
     def get_all_objects_by_type_id(self):
         try:
-            return object_manager.get_objects_by_type(self.public_id)
+            return object_manager.get_objects_by_type(int(self.public_id))
         except ObjectNotFoundError as e:
             return abort(400, e)
         except CMDBError:
             return abort(404)
 
     def get_type_by_id(self):
         try:
-            query = self._build_query({'public_id': self.public_id}, q_operator='$or')
+            query = self._build_query({'public_id': self.public_id})
             return object_manager.get_types_by(sort="public_id", **query)
         except TypeNotFoundError as e:
             return abort(400, e)
         except CMDBError:
             return abort(404)
 
-    #
-    # Advanced methods
-    #
-
-    def _build_query(self, args, q_operator='$and'):
+    @staticmethod
+    def _build_query(parameters, operator='$or') -> dict:
```

Comment:
This is the wrong doc style.
We are using: [Google Style Python Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)